runtime/near-vm-runner/src/prepare/prepare_v3.rs

```diff
@@ -448,7 +452,7 @@ impl finite_wasm_6::max_stack::SizeConfig for SimpleMaxStackCfg {
     }
 }
 
-struct SimpleGasCostCfg(u64);
+struct SimpleGasCostCfg(u64, u64, u64);
```

Comment:
Consider naming the fields here.

```suggestion
struct SimpleGasCostCfg {
    regular_op: u64, 
    linear_base: u64, 
    linear_element: u64
}
```

or something along those lines.